### ISSUE TITLE
Unify Model Binding

### ISSUE THREAD

**Issue by drusellers** [DarthFubuMVC/fubumvc#108]
Lo! and behold. I hereby summon the powers that be to provide for us, the mere mortals, a great and excellent grand unification of the Model Binding and Value Converting Families of amazing awesome with the most benevolent IObjectResolver. For too long our separate tribes have warred over the same terrain of watery code. I propose instead that we dry up the waters between our two camps and provide for us a direction of complete awesome.

TL;DR: please unify Model Binding / Value Conversion with IObjectResolver. :)

-d
